tests/Unit/Controller/AccountsControllerTest.php

```diff
@@ -438,6 +439,26 @@ public function testSendingError() {
 		$this->controller->send(13, 'sub', 'bod', 'to@d.com', '', '');
 	}
 
+	public function testSendingManyRecipientsError() {
+		$this->expectException(ManyRecipientsException::class);
+		$recipients = [];
+		for ($i = 0; $i < 11; $i++) {
+			$recipients[] = "$i@x.com";
+		}
+		$recipients = implode(',', $recipients);
+		$this->controller->send(13, 'sub', 'bod', $recipients, '', '');
+	}
+
+	public function testSendingManyRecipientsCcError() {
+		$this->expectException(ManyRecipientsException::class);
+		$recipients = [];
+		for ($i = 0; $i < 11; $i++) {
+			$recipients[] = "$i@x.com";
+		}
+		$recipients = implode(',', $recipients);
+		$this->controller->send(13, 'sub', 'bod', '', $recipients, '');
```

Comment:
please add an empty line before this

most of our tests follow the AAA pattern: arrange, act, assert. The first few lines are where you arrange things, the final one is where you act.